### PR TITLE
Use constant gmfDisplayQueryGridOptions

### PR DESCRIPTION
**Pull request by sbrunner**

### CODE REVIEW

File: contribs/gmf/src/options.js [hunk 1]
```diff
@@ -157,4 +162,97 @@
...
+ * @property {Stroke} [stroke] The stoke config.
+ * @property {Circle} [circle] The circle config.
+ * @property {number} [zIndex] The z index.
+ */
+
+/**
+ * @param {olStyle|Style} styleDescriptor The description of the style
+ * @returns {olStyle}
+ */
+export function buildStyle(styleDescriptor) {
```
Comment (ger-benjamin) [inline contribs/gmf/src/options.js:214]: @fredj Is there nothing to do that in ol ? Do you know ?

**Review by ger-benjamin — APPROVED** [review]
Looks good

File: contribs/gmf/src/options.js (hunk 1 above)
Comment (fredj) [reply contribs/gmf/src/options.js:214]: non, nothing like that in OpenLayers

Comment (ger-benjamin) [reply contribs/gmf/src/options.js:214]: Thanks (could be an useful tool)

**Pull request merged**